docs/developer-docs/smart-contracts/advanced-features/management-canister.mdx

```diff
@@ -0,0 +1,128 @@
+
+
+The address of the ICP management canister is `aaaaa-aa` (i.e. the empty blob). It is present on any ICP subnet. By using this address, a canister can  access the management canister either on the local or remote subnet -- ICP will take care of routing the request to the appropriate subnet transparently. The cost of making the call is charged to the calling canister just like any other inter-canister call.
+
+:::note
+
+Certain functionality like threshold signing and access to the bitcoin APIs is only enabled on certain subnets.
+
+:::
+
+The management canister can also be called by users. Most methods are only permitted to be called by the [controllers](https://internetcomputer.org/docs/current/developer-docs/smart-contracts/maintain/control/) of the canister. Certain calls can only be made by a canister and are not available for users.(e.g. `raw_rand` or `deposit_cycles`). When a user makes a call to the management canister, the cost of processing the request is charged to the canister that is being managed. 
+
+
+[Read more about the complete interface of the management canister.](https://internetcomputer.org/docs/current/references/ic-interface-spec#ic-candid)
+
+## Why does the management canister exist?
+
+You might be familiar with the [System API](https://internetcomputer.org/docs/current/references/ic-interface-spec#system-api) that allows canisters to interact with other canisters and perform special actions like sending messages or transferring cycles. A natural expectation would be to provide canister management through the same System API. However, the System API only works for synchronous calls that can be returned immediately. Some of the canister management operations are asynchronous in nature – the controlled canister might not even be in the same subnet as the controller. Therefore, an asynchronous API is needed.
+
+
+One option would be to build a custom asynchronous message passing protocol as an extension to the System API to support the canister management operations. This would require  re-implementing (to some extent) a lot of the mechanics that exist already to support ICP's message passing model through inter-canister calls. Providing this functionality through a virtual canister, i.e. the management canister, allows for ICP to reuse a lot of the mechanics that are already available and provides a familiar approach to users and canisters to access this API through inter-canister calls.
+
+## How can you interact with the management canister?
+
+Since the management canister behaves like a canister, users or other canisters can call its provided interfaces like they can do with any other canister on ICP.
```

Comment:
```suggestion
Since the management canister behaves like a canister, users or other canisters can call its provided interfaces like any other canister on ICP.
```